src/SonarScanner.MSBuild.PreProcessor/Interfaces/ISonarWebService.cs

```diff
@@ -61,7 +61,7 @@ public interface ISonarWebService : IDisposable
         /// <param name="targetDirectory">The directory to which the file should be downloaded</param>
         Task<bool> TryDownloadEmbeddedFile(string pluginKey, string embeddedFileName, string targetDirectory);
 
-        Task<AnalysisCacheMsg> DownloadCache(string projectKey, string branch);
+        Task<IList<SensorCacheEntry>> DownloadCache(string projectKey, string branch);
```

Comment:
Something to be considered, is if the SC and SQ caches will have different format, the naming of the method should contain if it is an SQ or SC cache, so that it is not used in the wrong spot.

Reply:
Indeed they will be different.

I think this is good for now since there is only a single implementation. I would wait for the next sprint to update the calls since then we will know more about the usage and we will be in a better position to update the naming and avoid the possibility to change this twice.